src/plugins/intel_npu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -685,17 +685,6 @@ std::vector<std::string> disabledTestPatterns() {
                 ".*smoke_OVClassNetworkTestP/OVClassNetworkTestPNPU.*"
         });
 
-        // [Tracking number: E#125086]
-        _skipRegistry.addPatterns(devices.has3720() && backendName.isZero(), 
-                "Failing tests after functional tests migration to OV", {
-                #ifdef WIN32
-                        ".*OVInferRequestPerfCountersExceptionTest.perfCountWereNotEnabledExceptionTest.*",
-                #elif defined(__linux__)
-                        ".*OVInferRequestMultithreadingTests.canRun3AsyncRequestsConsistently.*",
-                #endif
-                ".*OVCompiledModelPropertiesDefaultSupportedTests.CanCompileWithDefaultValueFromPlugin.*"
```

Comment:
It looks like CanCompileWithDefaultValueFromPlugin fails. Please fix it before merging it.